src/db/gc/snapshot.rs

```diff
@@ -239,6 +277,44 @@ where
                         }
                     }
                 }
+
+                // Backfill new db records during snapshot export
+                if let Ok(db) = open_db(self.db_root_dir.clone(), &self.db_config) {
+                    if let Some(mem_db) = self.memory_db.write().take() {
+                        let count = mem_db.len();
+                        let approximate_heap_size = {
+                            let mut s = 0;
+                            for (_k, v) in mem_db.iter() {
+                                s += 64;
+                                s += v.len();
+                            }
+                            s
+                        };
+                        let start = Instant::now();
+                        if let Err(e) = db.put_many_keyed(mem_db) {
+                            tracing::warn!("{e}");
+                        }
+                        tracing::info!(
```

Comment:
Let's use more meaningful variable names. Especially the meaning of `s` is not clear.

Reply:
Fixed.